Add SolarMax SMT

courvo opened the pull request:
new template for SolarMax SMT-series inverters.

based on senergy.yaml, but with scale factor of 0.1 for power and energy values.

courvo committed:
Create SolarMax SMT Series

new template for SolarMax SMT-series inverters.

based on senergy.yaml, but with scale factor of 0.1 for power and energy values.

andig commented:
Bitte Namenskonvention, Dateiendungen etc. wie bei der bestehenden SolarMax Implementierung. Ein Kommentar dass das eine Kopie mit anderer Skalierung ist wäre hilfreich.

courvo committed:
Update and rename SolarMax SMT Series to solarmax-inverter-smt

changed naming convention,
gernic description refers to SP inverters used as template, but with added scale factor of 0.1

andig commented on templates/definition/meter/solarmax-inverter-smt:
```diff
@@ -0,0 +1,31 @@
+products:
+  - brand: SolarMax
+    description:
+      generic: SolarMax SMT Series Inverter, based on  SP Series Inverter template, but with added scaling for power and energy values of 0.1
```
```suggestion
      generic: SolarMax SMT
```

andig commented on templates/definition/meter/solarmax-inverter-smt:
```diff
@@ -0,0 +1,31 @@
+template: solarmax-inverter-smt
```
```suggestion
template: solarmax-smt
```

andig commented on templates/definition/meter/solarmax-inverter-smt:
```diff
@@ -0,0 +1,31 @@
+    id: 1
+render: |
+  type: custom
+  {{- if eq .usage "pv" }}
```
Not needed if you have a single usage only

andig replied:
```suggestion
```

andig commented on templates/definition/meter/solarmax-inverter-smt:
```diff
@@ -0,0 +1,31 @@
+      type: holding
+      decode: uint32
+    scale: 0.1
+  {{- end }}
```
```suggestion
```

courvo committed:
Apply suggestions from code review

Co-authored-by: andig <cpuidle@gmail.com>

premultiply reviewed (commented):
Dem Template fehlt die .yaml-Endung
